<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModDetailPanel</class>
 <widget class="QWidget" name="ModDetailPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="basicInfoGroup">
     <property name="title">
      <string>基本信息</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>名称:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="nameValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="packageIdLabel">
        <property name="text">
         <string>Package ID:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="packageIdValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>作者:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="authorValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="versionLabel">
        <property name="text">
         <string>支持版本:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="versionValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="editableInfoGroup">
     <property name="title">
      <string>可编辑信息</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>类型:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="typeComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="remarkLabel">
        <property name="text">
         <string>备注:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="remarkTextEdit">
        <property name="maximumHeight">
         <number>100</number>
        </property>
        <property name="placeholderText">
         <string>输入备注信息...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>保存更改</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="descriptionGroup">
     <property name="title">
      <string>描述</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextBrowser" name="descriptionBrowser">
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dependenciesGroup">
     <property name="title">
      <string>依赖关系</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="dependenciesLabel">
        <property name="text">
         <string>依赖的 Mod:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="dependenciesList">
        <property name="maximumHeight">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loadBeforeLabel">
        <property name="text">
         <string>应在之前加载:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="loadBeforeList">
        <property name="maximumHeight">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loadAfterLabel">
        <property name="text">
         <string>应在之后加载:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="loadAfterList">
        <property name="maximumHeight">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="incompatibleLabel">
        <property name="text">
         <string>不兼容的 Mod:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="incompatibleList">
        <property name="maximumHeight">
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
